### PR TITLE
menus/dropdown respect theme

### PR DESCRIPTION
**Pull request by michaelkirk**
First commit is a no-op refactor.

## Night

Screen Shot 2021-02-24 at 5 51 09 PM

Screen Shot 2021-02-24 at 5 51 17 PM

## Legacy Day

Screen Shot 2021-02-24 at 5 51 37 PM

Screen Shot 2021-02-24 at 5 51 48 PM

## New Day

Screen Shot 2021-02-24 at 5 52 41 PM

Screen Shot 2021-02-24 at 5 52 54 PM

### CODE REVIEW

File: widgetry/src/widgets/mod.rs [hunk 1]
```diff
@@ -767,9 +766,6 @@ impl Widget {
         }
     }
 
-    pub(crate) fn take_menu<T: 'static + Clone>(self) -> Menu<T> {
```
Comment (dabreegster) [inline widgetry/src/widgets/mod.rs:770]: In some Java haze, apparently I thought downcasting made more sense than just having two constructors. Good cleanup.

**Comment by michaelkirk** [comment]
Actually, gonna tweak the design...

**Review by dabreegster — APPROVED** [review]
LGTM, thanks!

**Comment by dabreegster** [comment]
> Actually, gonna tweak the design...

Ah, I'll hold off review then. (Looked good from what I saw)

**Comment by michaelkirk** [comment]
Actually, going to re-open as is. I thought the contrast of the "hovered" item was a little intense, so I experimented with using the style from our "plain" buttons, but because the Menu bg is actually a little darker than the default panel there wasn't enough contrast IMO.

The better solution is probably somewhere in the middle, but I'm going to punt and just opt for "too much" contrast for now rather than too little.

Here's what that would have looked like FYI:
Screen Shot 2021-02-24 at 6 03 32 PM

Screen Shot 2021-02-24 at 6 04 45 PM
Screen Shot 2021-02-24 at 6 04 22 PM

**Comment by dabreegster** [comment]
Too much contrast SGTM -- I have to squint at those screenshots to see the active line.

LGTM, merging

**Pull request merged**